bluesky_queueserver/manager/plan_queue_ops.py

```diff
@@ -778,97 +777,97 @@ async def get_plan_history_size(self):
             The number of plans in the history.
         """
         async with self._lock:
-            return await self._get_plan_history_size()
+            return await self._get_history_size()
 
-    async def _get_plan_history(self):
+    async def _get_history(self):
         """
-        See ``self.get_plan_history()`` method.
+        See ``self.get_history()`` method.
         """
         all_plans_json = await self._r_pool.lrange(self._name_plan_history, 0, -1)
         return [json.loads(_) for _ in all_plans_json]
 
-    async def get_plan_history(self):
+    async def get_history(self):
         """
-        Get the list of all plans in the history. The first element of the list is
+        Get the list of all items in the plan history. The first element of the list is
         the oldest history entry.
 
         Returns
         -------
         list(dict)
-            The list of plans in the queue. Each plan is represented as a dictionary.
+            The list of items in the plan history. Each plan is represented as a dictionary.
             Empty list is returned if the queue is empty.
         """
         async with self._lock:
-            return await self._get_plan_history()
+            return await self._get_history()
 
-    async def _clear_plan_history(self):
+    async def _clear_history(self):
         """
-        See ``self.clear_plan_history()`` method.
+        See ``self.clear_history()`` method.
         """
-        while await self._get_plan_history_size():
+        while await self._get_history_size():
             await self._r_pool.rpop(self._name_plan_history)
 
-    async def clear_plan_history(self):
+    async def clear_history(self):
         """
         Remove all entries from the plan queue. Does not touch the running plan.
         The plan may be pushed back into the queue if it is stopped.
```

Comment:
```suggestion
        The item may be pushed back into the queue if it is stopped.
```

Reply:
This will remain true only for plans unless we decide to support executing Python scripts, which will not happen.